
libdir variable is not set correctly in generated $(STAGING_DIR)/usr/lib/libSDL.la
(prepended 2x with "$(STAGING_DIR)/usr").
This patch tries to correct that.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

--- buildroot.org/package/sdl/sdl.mk	2008-11-03 09:15:18.000000000 +0100
+++ buildroot/package/sdl/sdl.mk	2009-06-15 23:16:13.000000000 +0200
@@ -97,6 +97,9 @@
 # use correct rpath for linking
 	$(SED) 's^libdir=\$${exec_prefix}^libdir=/usr^' \
 		$(STAGING_DIR)/usr/bin/sdl-config
+# use correct libdir path in libSDL.la
+	$(SED) "s^libdir='$(STAGING_DIR)/usr^libdir='^" \
+		$(STAGING_DIR)/usr/lib/libSDL.la
 	touch -c $@
 
 $(TARGET_DIR)/usr/lib/libSDL.so: $(STAGING_DIR)/usr/lib/libSDL.so
